openedx/features/wikimedia_features/admin_dashboard/models.py

```diff
@@ -0,0 +1,179 @@
+"""
+WE'RE USING MIGRATIONS!
+
+If you make changes to this model, be sure to create an appropriate migration
+file and check it in at the same time as your model changes. To do that,
+
+1. Go to the edx-platform dir
+2. ./manage.py schemamigration admin_report_task --auto description_of_your_change
+3. Add the migration file created in edx-platform/lms/djangoapps/admin_report_task/migrations/
+
+
+ASSUMPTIONS: modules have unique IDs, even across different module_types
+
+"""
+import json
+import logging
+from uuid import uuid4
+
+from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
+from django.db import models, transaction
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext as _
+
+logger = logging.getLogger(__name__)
+
+# define custom states used by AdminReportTask
+QUEUING = 'QUEUING'
+PROGRESS = 'PROGRESS'
+TASK_INPUT_LENGTH = 10000
+
+
+@python_2_unicode_compatible
+class AdminReportTask(models.Model):
```

Comment:
can we use inheritance ?

Reply:
No, we cannot, we have to override a field(course_id) from the parent model(instructor_task). It will introduce additional complexity.